google/cloud/pubsub_v1/types.py

```diff
@@ -112,6 +119,11 @@
         "The maximum amount of time in seconds to hold a lease on a message "
         "before dropping it from the lease management."
     )
+    FlowControl.max_duration_per_lease_extension.__doc__ = (
+        "The max amount of time in seconds for a single lease extension attempt. "
+        "Bounds the delay before a message redelivery in the event the "
```

Comment:
super nit: replace "in the event" with "if"

Reply:
Done